src/cluster/sharding.rst

```diff
@@ -64,7 +64,9 @@ CouchDB nodes have a ``etc/default.ini`` file with a section named
 
 These settings specify the default sharding parameters for newly created
 databases. These can be overridden in the ``etc/local.ini`` file by copying the
-text above, and replacing the values with your new defaults.  The values can
+text above, and replacing the values with your new defaults.
+If ``[couch_peruser]`` ``q`` is set, that value is used for peruser databases.
+The values can
```

Comment:
I think there is a newline after `can` which can be removed and the section re-indented accordingly.

Should we also change `peruser` to `per-user` since it doesn't reference a config literal?